body/stretch_body/prismatic_joint.py

```diff
@@ -11,9 +11,11 @@ class PrismaticJoint(Device):
     """
     API to the Stretch Prismatic Joints
     """
-    def __init__(self,name):
+    def __init__(self,name,usb=None):
         Device.__init__(self,name )
-        self.motor = Stepper(usb='/dev/hello-motor-'+name)
+        if usb is None:
+            usb='/dev/hello-motor-'+name
```

Comment:
Instead of constructing the usb alias from `'/dev/hello-motor-'+name` here, let's make it a parameter of `name` in the robot_params_*_.py dictionaries. This gives users the ability to override in user YAML.

Reply:
It is a good idea and has been implemented with all the devices.